client/src/App.vue

```diff
@@ -35,6 +35,7 @@ function onLogoClick() {
       <div class="mb-2">
         <portal-target name="banner" />
       </div>
+      <portal-target name="modal" class="m-0" />
```

Comment:
I found out that Vue doesn't like it when you put the class label on portal targets. If you think the m-0 is necessary you can wrap it in a div (as I did above). Although since modals positions are position: fixed I would doubt margin will make any difference to the layout